### PR TITLE
client: pass permissions as a str rather than list

### PR DESCRIPTION
**Pull request by thetorpedodog**

### CODE REVIEW

File: tiledb/cloud/client.py [hunk 1]
```diff
@@ -5,8 +5,9 @@
 from . import config
 from . import tiledb_cloud_error
 from .rest_api import ApiException as GenApiException
-import urllib
 import os
+import typing as t
```
Comment (ihnorton) [inline tiledb/cloud/client.py:9]: Please use mypy style explicit imports.

Comment (ihnorton) [reply tiledb/cloud/client.py:9]: vscode's python extension can add them automatically (for better or sometimes worse)

**Review by ihnorton — CHANGES_REQUESTED** [review]
Style nit, but otherwise LGTM.

File: tiledb/cloud/client.py (hunk 1 above)
Comment (thetorpedodog) [reply tiledb/cloud/client.py:9]: Done.

Comment (ihnorton) [reply tiledb/cloud/client.py:9]: Sorry, I was unclear there -- by explicit imports I meant: `from typing import Union, Optional, ...`. That's the mypy convention and what we've adopted in other repos.

Comment (thetorpedodog) [reply tiledb/cloud/client.py]: ah jeez, I had not been staging my new changes when I ran `git commit --amend` so it was happily pushing no changes and just updating the commit time.

now **acutally** fixed.

Comment (ihnorton) [reply tiledb/cloud/client.py]: Thank you!

**Review by ihnorton — APPROVED** [review]

File: tiledb/cloud/client.py [hunk 2]
```diff
@@ -5,8 +5,8 @@
 from . import config
 from . import tiledb_cloud_error
 from .rest_api import ApiException as GenApiException
-import urllib
```
Comment (ihnorton) [inline tiledb/cloud/client.py:8]: I think this was unintentionally dropped here, which is causing the CI failure.

Comment (thetorpedodog) [reply tiledb/cloud/client.py:8]: Whoops! Fixed.

**Review by ihnorton — APPROVED** [review]

**Review by Shelnutt2 — APPROVED** [review]

**Pull request merged**